code/modules/mob/living/living.dm

```diff
@@ -839,12 +836,12 @@ Sorry Giacom. Please don't be mad :(
 
 	else if(istype(loc, /obj/machinery/atmospherics/components/unary/cryo_cell))
 		var/obj/machinery/atmospherics/components/unary/cryo_cell/C = loc
-		var/datum/gas_mixture/G = C.airs["a1"]
+		var/datum/gas_mixture/C_air_contents = C.airs[1]

```

Comment:
I'm not doing that git is and I hate it and have to change all that shit back manually now